src/preferences/dialog/dlgprefeffectsdlg.ui

```diff
@@ -470,30 +470,58 @@
     </widget>
    </item>
    <item>
-    <widget class="QGroupBox" name="groupMetaKnobOption">
+    <widget class="QGroupBox" name="groupEffectKnobOptions">
      <property name="title">
       <string>Effect load behavior</string>
      </property>
-     <layout class="QHBoxLayout" name="horizontalLayoutMetaKnob">
+     <layout class="QVBoxLayout" name="verticalLayoutEffectKnobOptions">
```

Comment:
Ideally, to keep "Keep.." and "Reset.." options aligned vertically, this would be a grid layout (i.e. no sub-layouts, just 4 items with explicit row & column attributes.

Tr strings only differ by one character (super vs. meta), so there's no significant misalignment to expect, though the perfectionist part of me prefers the grid layout ;)
@robbert-vdh If you don't have time to convert this let me know.